.github/actions/clp-core-build-containers/action.yaml

```diff
@@ -25,8 +19,6 @@ runs:
   using: "composite"
   steps:
     - uses: "docker/setup-buildx-action@v3"
-      with:
-        driver-opts: "network=host"
```

Comment:
Why does setup-buildx-action no longer need driver specified?

Reply:
This was necessary when we were pushing the image into the local registry. The local registry was running on the same machine running the container, so we needed `network=host` to access it; in contrast, ghcr.io is accessed over the internet.